docs/explanation/accessibility-architecture.md

```diff
@@ -49,3 +49,38 @@ A selection of applications on Ubuntu Desktop such as [App Center](https://githu
 | App Center               | Flutter       | [Flutter API](https://docs.flutter.dev/ui/accessibility-and-internationalization/accessibility) |
 | Software and Updates     | GTK3          | [ATK](https://docs.gtk.org/atk/) |
 
+
+## Screen reader and speech synthethis 
+
+Speech synthesizers converts text to human-sounding speech. Typically, they are offered as a backend for screen readers, but they can also be used as standalone programs. Orca is the default screen reader on Ubuntu Desktop. eSpeak and Speech Dispatcher are the 
+speech synthesizers that Orca depends on. When Orca identifies the text, it passes it to the Speech Dispatcher for speech synthesis; the Speech Dispatcher then passes the synthesized speech to eSpeak, which plays the sound.
+
+### Orca
+
+Orca is screen reading software for vision impaired and blind users. 
+
+Orca is preinstalled on Ubuntu Desktop and can be enabled and used out-of-box. To learn how to configure Orca, go to [Read the screen aloud](../how-to/accessibility/read-screen-aloud.md). 
+
+### BRLTTY
+
+The Orca screen reader can display the user interface on a refreshable Braille display. It uses the BRLTTY service, which provides access to the Linux console for a blind person using a refreshable Braille display. 
+
+As Orca, BRLTTY is preinstalled and can be enabled and used without any additional configurations, see [Read the screen in Braille](../how-to/accessibility/read-screen-in-braille.md).
+
+### eSpeak
+
+eSpeak is a speech synthesizer that supports English. eSpeak can be used as a standalalone tool or as a backend component for screen readers, see [eSpeak website](https://espeak.sourceforge.net/)
```

Comment:
eSpeak supports much more than English. Its language support is actually quite broad and that's the reason why it's the default:

https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages.md